util_test.go

```diff
@@ -0,0 +1,33 @@
+package gorepoman
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestCombineEnv(t *testing.T) {
+	orig := []string{
+		"A=1",
+		"B=2",
+		"C=3",
+		"D=",
+		"EFG=HIJKLMNOP",
```

Comment:
it'd be nice to have one test case where the value includes the `=` character

Reply:
https://github.com/fullstorydev/gorepoman/pull/9